File: python_backend/game_logic/ranges/range_funcs_special.py
from game_logic.ranges.pawn.top.pawn import pawn
from game_logic.ranges.king.top.king import king

__doc__ =\
    """
    range_funcs_special is range functions for special pieces, ones that can not be fully defined by the defs format
    """

range_funcs_special = {'Pawn': pawn, 'King': king}

File: node.js/ranges/rangeFuncsSpecial.js
var  pawn  = require("gameLogic/ranges/pawn/top/pawn");
var  king  = require("gameLogic/ranges/king/top/king");

/**
rangeFuncsSpecial is range functions for special pieces, ones that can ! be fully defined by the defs format
*/
let rangeFuncsSpecial = {'Pawn': pawn, 'King': king};

module.exports = rangeFuncsSpecial;

